scripts/v.unpack/v.unpack.py

```diff
@@ -216,9 +216,7 @@ def main():
         dbconn = grassdb.db_connection(force=True)
         todb = dbconn["database"]
         # return all tables
-        list_fromtable = grass.read_command(
-            "db.tables", driver="sqlite", database=fromdb
-        ).splitlines()
+        grass.read_command("db.tables", driver="sqlite", database=fromdb).splitlines()
```

Comment:
Any reason you kept this? I think this can be removed.

Reply:
I kept it because I thought it was there to cache errors. After deleting it and testing with testsuit, everything passed. It appears this line is unnecessary, as you guessed. I appreciate your thorough review.